README.rst

```diff
@@ -155,20 +154,16 @@ See docs/licenses for licenses of dependencies.
 .. |AppVeyorBuild| image:: https://ci.appveyor.com/api/projects/status/x4074ybuobsh4myx?svg=true
    :target: https://ci.appveyor.com/project/pygame/pygame
 
-.. |LaunchpadBuild| image:: https://www.pygame.org/images/launchpad_build.svg?svg=true
-   :target: https://code.launchpad.net/~pygame/+recipe/pygame-daily
-
 .. |PyPiVersion| image:: https://img.shields.io/pypi/v/pygame.svg?v=1
    :target: https://pypi.python.org/pypi/pygame
 
 .. |PyPiLicense| image:: https://img.shields.io/pypi/l/pygame.svg?v=1
    :target: https://pypi.python.org/pypi/pygame
 
-.. |Python2| image:: https://img.shields.io/badge/python-2-blue.svg?v=1
 .. |Python3| image:: https://img.shields.io/badge/python-3-blue.svg?v=1
 
 .. |GithubCommits| image:: https://img.shields.io/github/commits-since/pygame/pygame/2.0.2.svg
```

Comment:
This link should be updated to https://img.shields.io/github/commits-since/pygame/pygame/2.1.0.svg